src/main/java/com/volasoftware/tinder/controllers/FriendController.java

```diff
@@ -42,4 +43,20 @@ public ResponseEntity<?> getFriends(@Valid @RequestBody FriendSearchDto friendSe
             HttpStatus.OK,
             friendService.sortFriendsByLocation(friendSearchDto));
     }
+
+    @ApiOperation(value = "Get friend info", response = FriendDto.class)
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 200, message = "Success operation", response = FriendDto.class),
+            @ApiResponse(code = 401, message = "Unauthorized operation"),
+            @ApiResponse(code = 403, message = "Forbidden operation"),
+            @ApiResponse(code = 500, message = "Failed operation")
+        })
+    @GetMapping("/{id}")
+    public ResponseEntity<?> showFriendInfo(@Valid @PathVariable("id") Long accountId) {
```

Comment:
I see that you added `@Valid` annotation to PathVariable, what you want actually to get validated, also to make it work I believe that you should add `@Validated` on the class level.